### PR TITLE
✨ More error checks. Parse real extension name.

### PR DESCRIPTION
**Pull request by dimapaloskin**

### CODE REVIEW

**Comment by dimapaloskin** [comment]
@helsus Please look on it

File: .chromer/cli/bundle.js [hunk 1]
```diff
@@ -1,18 +1,42 @@
...
   try {
+    const distDir = resolve(process.cwd(), 'dist')
+    const builtManifestPath = resolve(distDir, 'manifest.json')
+    const keyPath = resolve(process.cwd(), 'keys/key.pem')
+
+    const isManifestExists = await pathExists(builtManifestPath)
+    const isKeyExists = await pathExists(keyPath)
+
+    if (!isManifestExists) {
+      console.error(`${chalk.bold('manifest.json')} does not exist. Run ${chalk.dim('yarn build')} before.`)
```
Comment (helsus) [inline .chromer/cli/bundle.js:19]: Perhaps move all these guard-alike procedures to the helper.js?

**Review by helsus — CHANGES_REQUESTED** [review]

File: .chromer/cli/bundle.js (hunk 1 above)
Comment (dimapaloskin) [reply .chromer/cli/bundle.js]: Don't really like 'helper' name, but I will doing something similar

Comment (dimapaloskin) [reply .chromer/cli/bundle.js:19]: I'm not sure that we really need to extract this method to separate module.
It's really non reusable methods with `bundle` context.  Also there is collision with `src/manifest.json` and `dist/manifest.json`. So it's will really strange if we are will move all file existing checks in the one file

Comment (helsus) [reply .chromer/cli/bundle.js:19]: Would you perhaps get rid of helper.js?

Comment (dimapaloskin) [reply .chromer/cli/bundle.js:19]: Will do, but not in this PR. I find useful functions like `assertNoFile` (rewrote `assertNoManifest`), but not for this PR ;)

**Review by helsus — APPROVED** [review]

**Pull request merged**